CONTRIBUTING.md

```diff
@@ -97,7 +97,7 @@ If a fix for the vulnerability isn't available or if it isn't practical to uptak
 
 This repository uses [Chromatic](https://www.chromatic.com) to facilitate visual component review, and adds GitHub status checks to the build pipeline. The workflow is as follows:
 
-1. The `UI Tests` status check is designed to highlight any visual changes included in the changeset. The developer (that's you!) should review the `UI Tests` status check in Chromatic, and if all changes are intentional or expected, mark the components as **approved**.
+1. The `UI Tests` status check is designed to highlight any visual changes included in the changeset. The developer (that's you!) should review the `UI Tests` status check in Chromatic, and if all changes are intentional or expected, mark the components as **approved**. Please wait until you're nearly ready to submit to do this; if you approve a snapshot and then the appearance changes as a result of PR feedback, the diffs can become confusing.
```

Comment:
I've noticed in the button appearance PRs that have gone through several rounds of feedback that it is hard to see a diff between main and the latest proposed changes if the diffs were approved from an intermediate state. I hope this will fix the problem.

Reply:
I updated this section as discussed in team meeting. We can make further changes to the UI Review policy if needed now that we understand its purpose better.